src/Microsoft.DocAsCode.Build.Engine/HostServiceCreatorWithIncremental.cs

```diff
@@ -111,5 +116,29 @@ private void PostCreate(HostService hostService, IEnumerable<FileAndType> files)
                 IncrementalContext.ReportModelLoadInfo(hostService, loadedFiles, BuildPhase.PreBuildBuild);
             }
         }
+
+        private bool ShouldProcessorTraceInfoCore(IDocumentProcessor processor)
```

Comment:
if a processor, like resourceProcessor, doesn't support incremental, we won't save any intermediate files for it, which is the name's meaning.